app/uk/gov/hmrc/hecstubs/models/DESResponse.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hecstubs.models
+
+import play.api.libs.json.JsValue
+
+final case class DESResponse(status: Int, responseBody: JsValue)
+
+object DESResponse {
+  val ctutrRes: String =
```

Comment:
these responses are specific to the get CTUTR API - if we ever need to integrate with another DES API these probably won't be applicable to them. Perhaps `GetCTUTRDESResponse`/ `GetCtutrDesResponse` would narrow the case class a bit?

Reply:
sure